graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -135,17 +142,45 @@ def write_parquet_to_relative_path(
 
     @staticmethod
     def create_new_relative_path_from_existing(
-        original_relative_path: str, repartitioned_file_index: int
+        original_relative_path: str, repartitioned_file_index: int, suffix: str = None
     ) -> str:
-        """
+        """Changes the index in the `original_relative_path` to `original_relative_path`.
+
         Given a path of the form 'path/to/parquet/part-00001-filename.snappy.parquet', changes the
         numerical part of the filename to match the provided `repartitioned_file_index`,
-        and changes the path prefix to 'path/to/parquet-repartitioned/'.
+        and changes the path prefix to 'path/to/parquet-repartitioned/', or
+        'path/to/parquet-repartitioned-{suffix}/' if `suffix` is provided.
```

Comment:
Can you move the examples in L176 here?

Reply:
It seems we will also add a uuid if suffix is not provided, but this is not mentioned in the docstring here.

Reply:
> It seems we will also add a uuid if suffix is not provided, but this is not mentioned in the docstring here.

Not sure if I understand correctly, the function has

```python
new_sub_path = (
            "parquet-repartitioned" if suffix is None else f"parquet-repartitioned-{suffix}"
        )
```

So unless a `suffix` is provided, only `parquet-repartitioned` is added. In our calls to the function in this file we do add a `uuid4()` suffix, but the function itself does not a suffix unless one is provided.

Reply:
> Can you move the examples in L176 here?

I'm trying to be consistent with Numpy docstring defined order that always has `Examples` as the last section of the docstring: https://numpydoc.readthedocs.io/en/latest/format.html#sections

Reply:
Now I c what you man regarding to the suffix.